src/components/Calculator.js

```diff
@@ -1,42 +1,67 @@
 import React, { Component } from 'react';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Please consider not disabling linters. They're there to help you

Reply:
Sorry @GzimAsani, but I had to do that because by default, ESLint expects functional components. But we are asked to do this with class components. 
The disabling of this linters will be removed in the next milestone, where I'm asked to refactor the project using functional components. I hope you understand.

Reply:
## OPTIONAL 
* Hi @Kingjosh007 you could use `PureCompnents` that way the linters won't give errors for example `class Bar extends React.PureComponent {
  render() {
    return <div>Baz</div>;
  }
}`

Reply:
Thanks @Davidosky007. Being new to React, I didn't know about PureComponents.